tests/find/test_find.cpp

```diff
@@ -532,7 +532,7 @@ void Test_OnlineAlgMulti(bool order_by_begin_position) {
         SEQAN_ASSERT_EQ(keywordIndex[8], 1u);
         SEQAN_ASSERT_EQ(finderPos[9], 6u);
         SEQAN_ASSERT_EQ(keywordIndex[9], 0u);
-    } else{                 
+    } else{
         SEQAN_ASSERT_EQ(finderPos[0], 0u);
```

Comment:
In this file you have many whitespace changes. Please undo. [and make a seperate PR if you want to, but then also against master]